libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsRoboTest.kt

```diff
@@ -438,21 +439,23 @@ class MapboxRouteLineUtilsRoboTest {
         assertEquals(
             33.3,
             (
-                (addStyleLayerSlots[20].contents as HashMap<String, Value>)["icon-offset"]
-                !!.contents as ArrayList<Value>
+                (addStyleLayerSlots[20].contents as HashMap<String, Value>)
+                ["icon-offset"]!!.contents as ArrayList<Value>
                 ).first().contents
         )
         assertEquals(
             44.4,
             (
-                (addStyleLayerSlots[20].contents as HashMap<String, Value>)["icon-offset"]
-                !!.contents as ArrayList<Value>
+                (
+                    addStyleLayerSlots[20].contents as HashMap<String, Value>
+                    )
```

Comment:
Let's fix the same way as in other locations:
```suggestion
                (addStyleLayerSlots[20].contents as HashMap<String, Value>)
```